lua/default-config.lua

```diff
@@ -14,23 +14,23 @@ lvim = {
   database = { save_location = "~/.config/lunarvim_db", auto_execute = 1 },
   keys = {},
 
-  -- TODO why do we need this?
+  -- Give autocompletion for builtins that lack a config function
```

Comment:
Comment needs to be reworked, was it a response to the previous one ?

Reply:
To give context.  Chris wasn't too involved with the callbacks to begin with.  And I merged the pr.  So I think he was confused about these changes.